spec/unit/mutant/actor/binding_spec.rb

```diff
@@ -1,12 +1,13 @@
 RSpec.describe Mutant::Actor::Binding do
-  let(:actor_a)    { double('Actor-A', sender: sender_a, receiver: receiver_a) }
-  let(:sender_a)   { double('Sender-A')                                        }
-  let(:sender_b)   { double('Sender-B')                                        }
-  let(:receiver_a) { double('Receiver-A')                                      }
-  let(:payload)    { double('Payload')                                         }
-  let(:type)       { double('Type')                                            }
+  let_instance(:sender_a,   Mutant::Actor::Sender                                           )
+  let_instance(:sender_b,   Mutant::Actor::Sender                                           )
+  let_instance(:mailbox,    Mutant::Actor::Mailbox, receiver: :receiver_a, sender: :sender_a)
```

Comment:
This syntax is also supported. The mapped ones get merged by the block ones.

I discovered that 99% of my let bindings can be handled without the explicit block at the call side. And its slightly less syntax in this form.